editor/app/ems/[incident_number]/page.tsx

```diff
@@ -183,23 +227,21 @@ export default function EMSDetailsPage({
           />
         </Col>
       </Row>
-      {matchingPeople && (
-        <Row>
-          <Col sm={12} className="mb-3">
-            <RelatedRecordTable<PeopleListRow, EMSLinkToPersonButtonProps>
-              records={matchingPeople}
-              isValidating={isValidating}
-              noRowsMessage="No crashes found"
-              header="Associated people records"
-              columns={emsMatchingPeopleColumns}
-              mutation=""
-              onSaveCallback={onSaveCallback}
-              rowActionComponent={EMSLinkToPersonButton}
-              rowActionComponentAdditionalProps={linkToPersonButtonProps}
-            />
-          </Col>
-        </Row>
-      )}
+      <Row>
+        <Col sm={12} className="mb-3">
+          <RelatedRecordTable<PeopleListRow, EMSLinkToPersonButtonProps>
+            records={matchingPeople ? matchingPeople : []}
+            isValidating={isValidating}
+            noRowsMessage="No people found"
```

Comment:
we already had a helpful no rows message in place but i needed to delete the `matchingPeople` check above for the table to still render with no people, also updated this message to say No people found instead of No crashes found